lib/moose/test_case/reporter.rb

```diff
@@ -25,6 +25,8 @@ def report!
             failure_script
           elsif test_case.passed?
             passed_script
+          else
```

Comment:
We don't want to assume the state of a test case falls into the general bucket. We likely  want to make this into an elsif